src/main/views/respondent-address-manual.njk

```diff
@@ -0,0 +1,31 @@
+{% extends "template.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {% include "back-link.njk"%}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      {% block errorSummary %}
+        {% include "./error-summary.njk" %}
+      {% endblock %}
+
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-4 ">
+       {{ h1 + respondentName }}?
+      </h1>
+      <p class="govuk-body">{{ pageText | safe}}</p>
```

Comment:
Suggest to remove | safe if not necessary

Reply:
removed